x-pack/plugins/integration_assistant/server/graphs/log_type_detection/prompts.ts

```diff
@@ -8,30 +8,27 @@ import { ChatPromptTemplate } from '@langchain/core/prompts';
 export const LOG_FORMAT_DETECTION_PROMPT = ChatPromptTemplate.fromMessages([
   [
     'system',
-    `You are a helpful, expert assistant in identifying different log types based on the format.
-
-Here is some context for you to reference for your task, read it carefully as you will get questions about it later:
-<context>
-<log_samples>
-{log_samples}
-</log_samples>
-</context>`,
+    `You are a helpful, expert assistant specializing in all things logs. You're great at analyzing log samples.`,
   ],
   [
     'human',
-    `Looking at the log samples , our goal is to identify the syslog type based on the guidelines below.
-Follow these steps to identify the log format type:
-1. Go through each log sample and identify the log format type.
+    `The current task is to identify the log format from the provided samples based on the guidelines below.
+
+The samples apply to the data stream {datastream_title} inside the integration package {package_title}.
+
+Follow these steps to do this:
+1. Go through each log sample and identify the log format. Output this as "name: <log_format>".
 2. If the samples have any or all of priority, timestamp, loglevel, hostname, ipAddress, messageId in the beginning information then set "header: true".
 3. If the samples have a syslog header then set "header: true" , else set "header: false". If you are unable to determine the syslog header presence then set "header: false".
-4. If the log samples have structured message body with key-value pairs then classify it as "log_type: structured". Look for a flat list of key-value pairs, often separated by spaces, commas, or other delimiters.
+4. If the log samples have structured message body with key-value pairs then classify it as "name: structured". Look for a flat list of key-value pairs, often separated by spaces, commas, or other delimiters.
 5. Consider variations in formatting, such as quotes around values ("key=value", key="value"), special characters in keys or values, or escape sequences.
-6. If the log samples have unstructured body like a free-form text then classify it as "log_type: unstructured".
-7. If the log samples follow a csv format then classify it as "log_type: csv".
-8. If the samples are identified as "csv" and there is a csv header then set "header: true" , else set "header: false".
-9. If you do not find the log format in any of the above categories then classify it as "log_type: unsupported".
+6. If the log samples have unstructured body like a free-form text then classify it as "name: unstructured".
+7. If the log samples follow a csv format then classify it with "name: csv". There are two sub-cases for csv:
+  a. If there is a csv header then set "header: true".
+  b. If there is no csv header then set "header: false" and try to find good names for the columns in the "columns" array by looking into the values of data in those columns. If you are unable to find good name for some column then keep it empty, like in the example.
```

Comment:
```suggestion
  b. If there is no csv header then set "header: false" and try to find good names for the columns in the "columns" array by looking into the values of data in those columns. If you are unable to find good name for any column then keep it empty, like in the example.
```

Reply:
Your version sounds to me like "if (for all columns you are unable to find a good name) then (...)" which would be different from what we want. I've tried to make the meaning even more clear with a slightly different wording.

Reply:
As long as LLM understands it I m Okay.